src/config/version.py

```diff
@@ -51,7 +51,9 @@ def is_prerelease_version(version):
 
 def read_release_version():
     try:
-        f = open("RELEASE-VERSION", "r")
+        import util
```

Comment:
I decided to install the python files to ```/usr/lib/python2.7/site-packages/``` and the ```res``` folder to ```/usr/share/fafclient```. 
This works pretty well because the ```util.COMMON_DIR``` variable is used everywhere (but not in the config/version.py script). Maybe this needs further refactoring and extraction of the paths from the util script.